### PR TITLE
Task/sdk 2255 lp wrapper

### PR DESCRIPTION
**Pull request by nzagorchev**
What              | Where/Who
------------------|----------------------------------------
JIRA | [SDK-2255](https://wizrocket.atlassian.net/browse/SDK-2255), [SDK-2382](https://wizrocket.atlassian.net/browse/SDK-2382)
People Involved   | @nzagorchev @hborisoff 

## Background
Support Leanplum migration to CleverTap.

## Implementation
- add `setInstanceWithAccountId` method
- use specific CleverTap instance
- bump CleverTap iOS version to 4.1.5

## Testing steps
Manual

### CODE REVIEW

**Review by hborisoff — APPROVED** [review]

**Comment by darshanclevertap** [comment]
@nzagorchev Change the PR to point to the `develop` branch first. We follow the `task` branch -> `develop` -> `master` flow

**Comment by akashvercetti** [comment]
@darshanclevertap  @nzagorchev We might need to bump the release sdk version in `package.json` too

**Comment by nzagorchev** [comment]
@akashvercetti @darshanclevertap Updated the version and changelog

**Review by akashvercetti — APPROVED** [review]

File: android/src/main/java/com/clevertap/react/CleverTapModule.java [hunk 1]
```diff
@@ -1373,28 +1373,44 @@ private <T> HashMap<String, T> eventPropsFromReadableMap(ReadableMap propsMap, C
...
+        clevertap.setInAppNotificationListener(this);
+        clevertap.setSyncListener(this);
+        clevertap.setCTNotificationInboxListener(this);
+        clevertap.setInboxMessageButtonListener(this);
+        clevertap.setCTInboxMessageListener(this);
+        clevertap.setInAppNotificationButtonListener(this);
+        clevertap.setDisplayUnitListener(this);
+        clevertap.setCTProductConfigListener(this);
+        clevertap.setCTFeatureFlagsListener(this);
+        clevertap.setLibrary("React-Native"); // TODO change library when using from Leanplum?
```
Comment (piyush-kukadiya) [inline android/src/main/java/com/clevertap/react/CleverTapModule.java:1387]: @hborisoff Do we need TODO here? If not can we remove please?

**Review by piyush-kukadiya — APPROVED** [review]
LGTM

**Pull request merged**